Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalModuleAlgo.cc

```diff
@@ -66,16 +66,16 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
   double waferGap = args.value<double>("waferGap");    // Gap between 2 wafers
   int sectors = args.value<int>("Sectors");            // Sectors
 #ifdef EDM_ML_DEBUG
-  edm::LogVerbatim("HGCalGeom") << "DDHGCalModule: zStart " << zMinBlock << " rFineCoarse " << rMaxFine
-                                << " wafer width " << waferW << waferW << " gap among wafers " << waferGap
-                                << " sectors " << sectors;
+  edm::LogVerbatim("HGCalGeom") << "DDHGCalModuleAlgo: zStart " << zMinBlock << " rFineCoarse " << rMaxFine
+                                << " wafer width " << waferW << " gap among wafers " << waferGap << " sectors "
+                                << sectors;
 #endif
-  const auto& slopeB = args.value<std::vector<double> >("SlopeBottom");   // Slope at the lower R
-  const auto& slopeT = args.value<std::vector<double> >("SlopeTop");      // Slopes at the larger R
-  const auto& zFront = args.value<std::vector<double> >("ZFront");        // Starting Z values for the slopes
-  const auto& rMaxFront = args.value<std::vector<double> >("RMaxFront");  // Corresponding rMax's
+  const auto& slopeB = args.value<std::vector<double> >("SlopeBottom");  // Slope at the lower R
+  const auto& slopeT = args.value<std::vector<double> >("SlopeTop");     // Slopes at the larger R
+  auto zFront = args.value<std::vector<double> >("ZFront");              // Starting Z values for the slopes
```

Comment:
why are these now copies rather than const refs? It doesn't seem like the values are changed anywhere later.